server/server/src/lib.rs

```diff
@@ -88,7 +88,7 @@ async fn run_stage0(
     {
         service_provider
             .app_data_service
-            .set_hardware_id(uuid().to_ascii_uppercase())?;
+            .set_hardware_id(machine_uid::get().unwrap())?;
```

Comment:
For android - the ANDROID_ID is passed in to `Java_org_openmsupply_client_RemoteServer_startServer` - could you pass that in to the server and save as well please 🙇 

Currently the settings object is constructed and passed to the server - you could add in to that maybe?